server/configs/config.json

```diff
@@ -3,6 +3,7 @@
     "username": "PG_USERNAME",
     "password": "PG_PASSWORD",
     "database": "PG_DATABASE",
+    "dialect": "postgres",
```

Comment:
I updated the config to include the dialect at the top level because I was getting an error `Dialect needs to be explicitly supplied as of v4.0.0` while running migrations. Adding this line to the config fixed it.